orc8r/cloud/go/services/ctraced/servicers/trace_servicer.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package servicers
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"magma/orc8r/cloud/go/orc8r"
+	"magma/orc8r/cloud/go/serdes"
+	"magma/orc8r/cloud/go/services/configurator"
+	"magma/orc8r/cloud/go/services/ctraced/obsidian/models"
+	"magma/orc8r/cloud/go/services/ctraced/storage"
+	merrors "magma/orc8r/lib/go/errors"
+	"magma/orc8r/lib/go/protos"
+)
+
+type CallTraceServer struct {
+	storage storage.CtracedStorage
+}
+
+func NewCallTraceServer(storage storage.CtracedStorage) *CallTraceServer {
+	return &CallTraceServer{storage: storage}
+}
+
+func (srv *CallTraceServer) ReportEndedCallTrace(ctx context.Context, req *protos.ReportEndedTraceRequest) (*protos.Void, error) {
```

Comment:
General: use full, unique service message names, not void -- https://magma.github.io/magma/docs/next/contributing/contribute_conventions#grpc